letsencrypt/README.md

```diff
@@ -14,3 +14,10 @@ The generated certificate can be used within others addons. By default the path
 [issue]: https://github.com/home-assistant/hassio-addons/issues
 [certbot]: https://certbot.eff.org
 [reddit]: https://reddit.com/r/homeassistant
+
+[Almond]: https://almond.stanford.edu/
+[aarch64-shield]: https://img.shields.io/badge/aarch64-yes-green.svg
+[amd64-shield]: https://img.shields.io/badge/amd64-yes-green.svg
+[armhf-shield]: https://img.shields.io/badge/armhf-no-red.svg
+[armv7-shield]: https://img.shields.io/badge/armv7-yes-green.svg
+[i386-shield]: https://img.shields.io/badge/i386-no-red.svg
```

Comment:
I believe this supports armhf and i386...

Reply:
This ref seems to be used. You mean we should merge to shields?

Reply:
I mean that for armhf and i386 it now shows a red badge with "no" in it, indicating it is not supported, while this add-on does support it.